File: UBL/CommonAggregateComponents/ResultOfVerification.cs
﻿using System.Xml.Linq;
using Hilma.UBL.Attributes;
using Hilma.UBL.UnqualifiedDataTypes;

namespace Hilma.UBL.CommonAggregateComponents
{
  [Contract]
  public class ResultOfVerification
  {
    public IdentifierType ValidatorID { get; set; }
    public CodeType ValidationResultCode { get; set; }
    public DateType ValidationDate { get; set; }
    public TimeType ValidationTime { get; set; }
    public string ValidateProcess { get; set; }
    public string ValidateTool { get; set; }
    public string ValidateToolVersion { get; set; }
    public Party SignatoryParty { get; set; }

  }

}

File: Espd/typings/ResultOfVerification.ts
import { IdentifierType } from './IdentifierType'
import { CodeType } from './CodeType'
import { DateType } from './DateType'
import { TimeType } from './TimeType'
import { Party } from './Party'
import assign from 'lodash.assign'

export class ResultOfVerification { 
    public constructor(init?:Partial<ResultOfVerification>) {
        assign(this, init)
    } 
    
    
    validatorID?: IdentifierType
    validationResultCode?: CodeType
    validationDate?: DateType
    validationTime?: TimeType
    validateProcess?: string
    validateTool?: string
    validateToolVersion?: string
    signatoryParty?: Party
}
